doc/sphinx/locale/es/LC_MESSAGES/installation.po

```diff
@@ -378,8 +378,8 @@ msgid ""
 msgstr "Puedes instalar cuantas instancias desees de cada perfil, pero ten "
 "encuenta que, mientras algunos estan pensados para escalar "
```

Comment:
Although github doesn't show it, this has been fixed in https://github.com/irontec/ivozprovider/pull/40/commits/e80717d59516934cc354d9aa2a36ebe0566ee9af